HARK/distribution.py

```diff
@@ -57,6 +57,20 @@ def __init__(self, mu = 0.0, sigma = 1.0, seed = 0):
         # Set up the RNG
         super().__init__(seed)
 
+        if isinstance(self.mu, list) != isinstance(self.sigma, list):
+            raise Exception(
+                "mu and sigma must be of same types. They are %s, %s" % (
+                    type(self.mu), type(self.sigma)
+                    ))
+
+        if isinstance(self.mu, list):
+            if len(list(self.mu)) != len(list(self.sigma)):
+                raise Exception(
+                    "mu and sigma must be of same length, are %d, %d" % (
+                        len(list(self.mu)), len(list(self.sigma))
+                    ))
+
```

Comment:
what should happen if there are not lists?

Reply:
Like, if they are scalars?  Then they should be converted into length-1 lists, I guess ...

Reply:
The internal representation should be numpy arrays, not lists.  If the user provides a one-dimensional input, mu and sigma, those should be converted to two 1-element np arrays.

These topics should be moved to a different issue.  @MridulS or @sbenthall will create the new issue